microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -261,11 +261,17 @@ Issued: 13 August 2024
 
 For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
 
-[id="microshift-4-16-8-dp_{context}"]
-=== RHBA-2024:5424 - {microshift-short} 4.16.8 bug fix and enhancement advisory
-
 Issued: 20 August 2024
 
 {product-title} release 4.16.8 is now available. The list of bug fixes that are included in the update is attached to the link:https://access.redhat.com/errata/RHBA-2024:5424[RHBA-2024:5424] advisory. The images that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:5422[RHSA-2024:5422] advisory.
 
 For the latest images included with {microshift-short}, view the contents of the `microshift-release-info` RPM. See xref:../microshift_install/microshift-embed-in-rpm-ostree-offline-use.adoc#microshift-embed-microshift-image-offline-deployment_microshift-embed-rpm-ostree-offline-use[Embedding {microshift-short} containers for offline deployments].
+
+[id="microshift-4-16-9-dp_{context}"]
```

Comment:
Kind of outside this update but I noticed the heading for 4.16.8 is missing. Raw file: 

![Screenshot from 2024-08-26 15-18-31](https://github.com/user-attachments/assets/731154cf-177f-4f0f-ba7f-375540ca99d9)

Reply:
Breaking all kinds of things I already fixed today. Gah